tests/integration_tests/test_chainermn.py

```diff
@@ -304,6 +304,37 @@ def test_init(storage_mode, comm):
 
             assert mn_trial.number == trial.number
 
+    @staticmethod
+    @pytest.mark.parametrize("storage_mode", STORAGE_MODES)
+    def test_suggest_float(storage_mode, comm):
```

Comment:
How about also checking that exceptions are raised as excepted similar to other tests below?

Reply:
Thanks!
I applied your suggested change.

Reply:
Thanks for your quick action.